drivers/video/video_mcux_csi.c

```diff
@@ -404,7 +404,9 @@ static const struct video_driver_api video_mcux_csi_driver_api = {
 #if 1 /* Unique Instance */
 static const struct video_mcux_csi_config video_mcux_csi_config_0 = {
 	.base = (CSI_Type *)DT_INST_REG_ADDR(0),
+#if DT_INST_NODE_HAS_PROP(0, sensor_label)
 	.sensor_label = DT_INST_PROP(0, sensor_label),
```

Comment:
This seems unrelated to this feature, and should perhaps be part of a PR that changes the property from a label to a phandle.  For the purposes of this PR it's OK to keep it this way.

Reply:
I've added the change in this PR